substrate/frame/identity/src/lib.rs

```diff
@@ -1024,13 +1070,13 @@ pub mod pallet {
 			let authority = T::Lookup::lookup(authority)?;
 			// We don't need to check the length because it gets checked when casting into a
 			// `BoundedVec`.
-			Self::validate_username(&suffix, None).map_err(|_| Error::<T>::InvalidSuffix)?;
+			Self::validate_suffix(&suffix)?;
 			let suffix = Suffix::<T>::try_from(suffix).map_err(|_| Error::<T>::InvalidSuffix)?;
 			// The authority may already exist, but we don't need to check. They might be changing
 			// their suffix or adding allocation, so we just want to overwrite whatever was there.
```

Comment:
Comment needs to be updated.
Do we still don't need to check for overwrite? This might be overwriting authority for a given suffix.

Reply:
I'll rethink this section.